instrumentation/github.com/aws/aws-sdk-go/service/otels3/config.go

```diff
@@ -13,63 +13,63 @@
 // limitations under the License.
 
 // Package config sets options for the aws sdk instrumentation
-package config
+package otels3
 
 import (
 	"go.opentelemetry.io/otel"
 	otelmetric "go.opentelemetry.io/otel/api/metric"
 	oteltrace "go.opentelemetry.io/otel/api/trace"
 )
 
-// Config provides options for the aws sdk instrumentation.
-type Config struct {
-	TracerProvider           oteltrace.TracerProvider
-	MetricProvider           otelmetric.MeterProvider
-	Propagators              otel.TextMapPropagator
-	SpanCorrelationInMetrics bool
+// Config provides options for the AWS SDK instrumentation.
+type config struct {
+	TracerProvider  oteltrace.TracerProvider
+	MetricProvider  otelmetric.MeterProvider
+	Propagators     otel.TextMapPropagator
+	SpanCorrelation bool
 }
 
 // Option interface used for setting instrumentation configuration options.
 type Option interface {
-	Apply(*Config)
+	apply(*config)
 }
 
 // OptionFunc provides a wrapper for specifying options in function format
-type OptionFunc func(*Config)
+type optionFunc func(*config)
 
 // Apply will set the option in the provided config.
```

Comment:
we also need to update the comments to lower case

Reply:
will fix this one